modules/statistics/php/stats_reliability.class.inc

```diff
@@ -125,12 +129,8 @@ class Stats_Reliability extends \NDB_Form
         // Whether something is reliable or not isn't stored in the database,
         // but calculated on the fly.
         // As a result, we need to manually calculate it and maintain for each
-        // type of thing Tanya wants stats for here separately
-
-        // Thresholds was copy/pasted from
-        include_once __DIR__ .
-            "/../../reliability/php/NDB_Menu_Filter_reliability.class.inc";
-        $thresholds = \NDB_Menu_Filter_Reliability::getThreshholds();
+        // type of thing.
+        $thresholds = Reliability::getThreshholds();
```

Comment:
Can you make a ticket to move this function so that this require_once isn't required?